CONTRIBUTING.md

```diff
@@ -116,6 +117,35 @@ Use your real name (sorry, no pseudonyms or anonymous contributions.)
 If you set your `user.name` and `user.email` git configs, you can sign your
 commit automatically with `git commit -s`.
 
+## Merge bot interaction
+
+Maintainers should never merge anything directly into upstream
+branches.  Instead, interact with the [bors-ng bot](https://bors.tech/)
+through PR comments as summarized below. This ensures all upstream
+branches contain commits in a predictable order, and that every commit
+has passed automated testing at some point in the past. A
+[Maintainer portal](https://app.bors.tech/repositories/22803)
+is available, showing all PRs awaiting review and approval.
+
+Common [bors-ng comment commands](https://bors.tech/documentation/):
```

Comment:
These commands are to be added by themselves in a line in a comment of the PR?  Do you want to state something to that effect?

Reply:
good idea.  Got it.